pymtl3/passes/PassGroups.py

```diff
@@ -1,88 +1,56 @@
 from pymtl3.dsl import Component
 
-from .CLLineTracePass import CLLineTracePass
-from .DynamicSchedulePass import DynamicSchedulePass
-from .GenDAGPass import GenDAGPass
-from .LineTraceParamPass import LineTraceParamPass
-from .mamba.HeuristicTopoPass import HeuristicTopoPass
-from .mamba.TraceBreakingSchedTickPass import TraceBreakingSchedTickPass
-from .mamba.UnrollTickPass import UnrollTickPass
-from .OpenLoopCLPass import OpenLoopCLPass
-from .SimpleSchedulePass import SimpleSchedulePass
-from .SimpleTickPass import SimpleTickPass
-from .text_wave.CollectSignalPass import CollectSignalPass
-from .text_wave.PrintWavePass import PrintWavePass
-from .VcdGenerationPass import VcdGenerationPass
-from .WrapGreenletPass import WrapGreenletPass
-
-SimpleSim = [
-  Component.elaborate,
-  GenDAGPass(),
-  WrapGreenletPass(),
-  SimpleSchedulePass(),
-  CLLineTracePass(),
-  # VcdGenerationPass(),
-  CollectSignalPass(),
-  PrintWavePass(),
-  SimpleTickPass(),
-  LineTraceParamPass(),
-  Component.lock_in_simulation
-]
-
-DynamicSim = [
-  Component.elaborate,
-  GenDAGPass(),
-  WrapGreenletPass(),
-  DynamicSchedulePass(),
-  CLLineTracePass(),
-  SimpleTickPass(),
-  Component.lock_in_simulation
-]
+from .autotick.OpenLoopCLPass import OpenLoopCLPass
+from .BasePass import BasePass
+from .sim.DynamicSchedulePass import DynamicSchedulePass
+from .sim.GenDAGPass import GenDAGPass
+from .sim.SimpleSchedulePass import SimpleSchedulePass
+from .sim.SimpleTickPass import SimpleTickPass
+from .sim.WrapGreenletPass import WrapGreenletPass
+from .tracing.CLLineTracePass import CLLineTracePass
+from .tracing.CollectSignalPass import CollectSignalPass
+from .tracing.LineTraceParamPass import LineTraceParamPass
+from .tracing.PrintWavePass import PrintWavePass
+from .tracing.VcdGenerationPass import VcdGenerationPass
+
+
+# SimpleSim can be used when the UDG is a DAG
+class SimpleSimPass( BasePass ):
```

Comment:
Now that each "PassGroup" is actually a pass, I wonder whether we should keep the name "PassGroup" or not... Is "Passes" better than "PassGroups"? What do you think?

Reply:
Hmmm I mean it's basically a "top-level pass" or "a sequence of passes that serve a purpose"

Reply:
@cbatten comments on this are welcome